attributes/default.rb

```diff
@@ -6,9 +6,6 @@
                                  '386'
                                end
 
-default['terraform']['zipfile'] = "terraform_#{node['terraform']['version']}_" \
```

Comment:
Agreed.  For those self-hosting the zipfile in a custom location this is a handy attribute.  

If I recall the issue I was having, with the way it is now, this zipfile generation logic only works if you don't try and override the version being set in this attributes file. 

I'll revisit this and try to keep the attribute setting option (and the default as it is set now) while correctly generating the path to a zip file when setting the version to something other than what the cookbook defaults to.